python/pyspark/daemon.py

```diff
@@ -177,4 +183,25 @@ def handle_sigterm(*args):
 
 
 if __name__ == '__main__':
-    manager()
```

Comment:
I think:

https://github.com/apache/spark/blob/87ffe7adddf517541aac0d1e8536b02ad8881606/python/test_coverage/coverage_daemon.py#L45

should be updated too BTW.